clients/extension/src/pages/plugin/index.tsx

```
@@ -0,0 +1,62 @@
+import { EmailProvider } from '@core/ui/state/email'
+import { PasswordProvider } from '@core/ui/state/password'
+
+import { initializeMessenger } from '../../messengers/initializeMessenger'
+import { ServerPasswordStep } from '@core/ui/mpc/keygen/create/fast/server/password/ServerPasswordStep'
+import { useStepNavigation } from '@lib/ui/hooks/useStepNavigation'
+import { useNavigateBack } from '@lib/ui/navigation/hooks/useNavigateBack'
+import { Match } from '@lib/ui/base/Match'
+import { ServerEmailStep } from '@core/ui/mpc/keygen/create/fast/server/email/ServerEmailStep'
+import { ReshareFastKeygenServerActionProvider } from '@core/ui/mpc/keygen/reshare/ReshareFastKeygenServerActionProvider'
+import { StepTransition } from '@lib/ui/base/StepTransition'
+import { FastKeygenServerActionStep } from '@core/ui/mpc/keygen/fast/FastKeygenServerActionStep'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)